app/src/main/java/org/mifos/selfserviceapp/ui/fragments/LoanAccountSummaryFragment.java

```diff
@@ -0,0 +1,152 @@
+package org.mifos.selfserviceapp.ui.fragments;
+
+import android.app.ProgressDialog;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.models.accounts.loan.LoanAccount;
+import org.mifos.selfserviceapp.presenters.LoanAccountsDetailPresenter;
+import org.mifos.selfserviceapp.ui.activities.base.BaseActivity;
+import org.mifos.selfserviceapp.ui.views.LoanAccountsDetailView;
+import org.mifos.selfserviceapp.utils.Constants;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by dilpreet on 25/2/17.
+ */
+
+public class LoanAccountSummaryFragment extends Fragment implements LoanAccountsDetailView {
+
+    @BindView(R.id.tv_loan_product_name)
+    TextView tvLoanProductName;
+
+    @BindView(R.id.tv_principal)
+    TextView tvPrincipalName;
+
+    @BindView(R.id.tv_interest_charged)
+    TextView tvInterestChargedName;
+
+    @BindView(R.id.tv_fees)
+    TextView tvFeesName;
+
+    @BindView(R.id.tv_penalties)
+    TextView tvPenaltiesName;
+
+    @BindView(R.id.tv_total_repayment)
+    TextView tvTotalRepaymentName;
+
+    @BindView(R.id.tv_total_paid)
+    TextView tvTotalPaidName;
+
+    @BindView(R.id.tv_interest_waived)
+    TextView tvInterestWaivedName;
+
+    @BindView(R.id.tv_penalties_waived)
+    TextView tvPenaltiesWaivedName;
+
+    @BindView(R.id.tv_fees_waived)
+    TextView tvFeesWaivedName;
+
+    @BindView(R.id.tv_outstanding_balance)
+    TextView tvOutstandingBalanceName;
+
+    @Inject
+    LoanAccountsDetailPresenter mLoanAccountsDetailPresenter;
+
+    private long loanId;
+
+    private View rootView;
+
+    private ProgressDialog progressDialog;
```

Comment:
We are making GET request. So there is not the point of losing any data if the user will switch to a different screen. 
Remove the ```ProgressDialog``` and use the ```ProgressBar``` Instead in XML.